docs/resource/os.md

```diff
@@ -15,31 +15,46 @@ In case of virtualized environments, this is the operating system as it is obser
+| `linux`        | Linux                                | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `netbsd`       | NetBSD                               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `openbsd`      | OpenBSD                              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `solaris`      | SunOS, Oracle Solaris                | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `windows`      | Microsoft Windows                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `z_os`         | IBM z/OS                             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
+## OS Build ID Lookup
 
+To standardize the retrieval of OS build ID information across different operating systems, SDK detector implementations should use the sources listed below.
+
+| OS      | Primary                                                                                              | Fallback                                                                      |
+| ------- | ---------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------- |
+| Windows | `CurrentBuildNumber` from registry `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion` | -                                                                             |
+| MacOS   | `ProductBuildVersion` from `/System/Library/CoreServices/SystemVersion.plist`                        | `ProductBuildVersion` from `/System/Library/CoreServices/ServerVersion.plist` |
+| Linux   | `BUILD_ID` from `/etc/os-release`                                                                    | `BUILD_ID` from `/usr/lib/os-release`                                         |
+|         |                                                                                                      | contents of `/proc/sys/kernel/osrelease`                                      |
+
+### Implementation Notes
+
+1. **Windows**: The build ID can be retrieved from the registry key `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion`, where the `CurrentBuildNumber` value holds the desired information.
+2. **MacOS**: On MacOS, the build ID is stored in the `ProductBuildVersion` field within the plist files located at `/System/Library/CoreServices/SystemVersion.plist` and `/System/Library/CoreServices/ServerVersion.plist`. The first file is typically used for regular MacOS installations, while the second might be used in server configurations.
+3. **Linux**: Linux distributions typically store the build ID in the `/etc/os-release` or `/usr/lib/os-release` files under the `BUILD_ID` field. If these files or the `BUILD_ID` field are not available, the `os.build_id` can be obtained from the `/proc/sys/kernel/osrelease` file as a fallback.
```

Comment:
This recommendation should probably be part of the yaml file and rendered automatically. https://github.com/open-telemetry/semantic-conventions/blob/main/model/resource/os.yaml#L15

Reply:
Could you provide the file with the same implementation? I can't find it in the resources host.

Reply:
Look for `notes: ` in the yaml files, like this https://github.com/open-telemetry/semantic-conventions/blob/main/model/trace/database.yaml#L259

markdown syntax should work there as well.

Reply:
@joaopgrassi fixed